src/status_im/ui/screens/intro/views.cljs

```diff
@@ -62,7 +62,8 @@
                                 :justify-content :flex-end
                                 :align-items :center
                                 :padding-horizontal 32}}
-            (let [margin 32
+            (let [height (:atom (first (filterv #(= (:key %) (:title s)) heights)))
```

Comment:
im not sure if we want to set different sizes for images @errorists  wdyt? my suggestion is just to use lower height for all . something like
```clojure
(let [new-height (-> e .-nativeEvent .-layout .-height)]
     (swap! height #(if (pos? %) (min % new-height) new-height))
```

Reply:
different sizes for images? like you mean, in a single carousel on a signle device, they could have different dimensions? oh for sure **not**!!

Reply:
@yenda @errorists, so I had this initially and it means that there is a large gap in those slides with only one line of text. The issue is that some slides have multiple lines of title text and some slides *do not*. For example, the first slide has two lines and the subsequent slides have one. So what ends up happening is the minimum image height is calculated, and there ends up being a large gap between the image in slides 2 and 3. Are you sure this is what you want?

Reply:
ping @errorists

Reply:
there's already a larger gap between the image and title in the design. 
☝️ image aligns to the top, 
👇 Text floats to the bottom, 
with the space between them having at least 16pt in height, otherwise we start to scale the image down

![Screenshot 2020-04-28 at 10 35 42](https://user-images.githubusercontent.com/4932774/80466137-655f2a00-893c-11ea-9d3e-d3ede62d8608.png)

Reply:
> Are you sure this is what you want?

looks like yes :)

Reply:
OK, all updated. Here's what it looks like on a narrow device:
<img src="https://imgur.com/ZzaH5YR.png" width="200"><img src="https://imgur.com/TXG5USi.png" width="200"><img src="https://imgur.com/3aMPgud.png" width="200">